### ISSUE TITLE
Leptos fork no longer needed

### ISSUE THREAD

**Issue by DylanRJohnston** [9oelM/leptos-cf#5]
The dependencies for this project can be reverted back to the upstream repository as the changes you made in the fork are now in the upstream.

**Comment by 9oelM** [comment]
> The dependencies for this project can be reverted back to the upstream repository as the changes you made in the fork are now in the upstream.

Yes that is correct. However the upstream is now at ~0.5.0, while leptos-cloudflare only supports leptos@0.4.9. This means we will have to wait until I introduce changes to this project to support ~0.5.0.

I will leave this issue open to track the status of the support for leptos@~0.5.0.

**Comment by DylanRJohnston** [comment]
I actually did this today, but I've removed the serving of static assets from the KV because I couldn't figure out how to update that correctly. Instead I've added support for Pages. https://github.com/xrpl-mm/leptos-cloudflare/pull/6

**Comment by webbrain-one** [comment]
Hi! I've opened a pull request to revert the dependencies back to the upstream repository, as the changes from the fork have now been merged upstream.

PR: https://github.com/9oelM/leptos-cloudflare/pull/8
